Formula/yarp.rb

```diff
@@ -3,17 +3,17 @@ class Yarp < Formula
   homepage "http://yarp.it"
 
   stable do
-    url "https://github.com/robotology/yarp/archive/v2.3.70.1.tar.gz"
-    sha256 "c693956aa76082bff18645a4a0cadf43d564c82fe518420e2b6c8410f716bf85"
+    url "https://github.com/robotology/yarp/archive/v2.3.70.2.tar.gz"
+    sha256 "888c4d2d85e0d9a3da4c4a111ed96c77c35a2849ed9bf079d69b08941f40ea79"
   end
 
   bottle do
-    root_url "https://github.com/robotology/yarp/releases/download/v2.3.70.1"
-    sha256 "f932056d3f1b2cac2cad6c8695f807e142bea0f1927e4507ec616ef5524ccbc9" => :sierra
+    root_url "https://github.com/robotology/yarp/releases/download/v2.3.70.2"
+    sha256 "d68bd7e09a45f3b066f1556c8016d3ad3b1f20ebd1c2b4868a75171aa14c0564" => :sierra
   end
 
   head do
-      url "https://github.com/robotology/yarp.git", :branch => "devel"
+      url "https://github.com/robotology/yarp.git", :branch => "master"
```

Comment:
I agree on having `head` pointing to `master`, but last time we decided to change it to `devel`.
If for you is ok, it is ok also for me :)

Reply:
Discussing with @traversaro we said that it is actually better to have master as otherwise the formula with --head just won't work.

Reply:
Why should it not work?

Reply:
The problem is with `robot-testing` that is installed from master branch and clasehs with YARP devel.

Reply:
More in general, typically every robotology project in the `devel` branch depends on the `devel` features of its robotology dependency (and this is good), but homebrew always install the released version of the dependency of a formula, unless a user manually installs all the robotology projects with the `--HEAD` options all the dependencies.

Reply:
I hoped it would have been possible to depend on the `head` version of another formula, but probably this is not the case

Reply:
It doesn't seems to straightfroward to do it, but probably with a bit convoluted rb code we can make it happen. That's why I asked for the `resource` in the Wiki.